src/main/java/carleton/sysc4907/DiagramEditorLoader.java

```diff
@@ -9,15 +9,22 @@
 import carleton.sysc4907.controller.*;
 import carleton.sysc4907.controller.element.*;
 import carleton.sysc4907.model.*;
+import carleton.sysc4907.processing.ElementCreator;
+import carleton.sysc4907.processing.FontOptionsFinder;
 import javafx.scene.Scene;
 import javafx.stage.Stage;
+import org.xml.sax.SAXException;
 
+import javax.xml.parsers.ParserConfigurationException;
 import java.io.IOException;
+import java.net.URISyntaxException;
 import java.util.LinkedList;
 import java.util.List;
 
 public class DiagramEditorLoader {
 
+    private final String TEMPLATE_FILE_PATH = "/carleton/sysc4907/templates.xml";
```

Comment:
That's a good point. Since it's mostly unrelated to this PR and it's not priority (doesn't affect behavior), I think it's best to create a separate issue for the config file, but it's definitely something to look into.